spec/controllers/orgs/rosters_controller_spec.rb

```diff
@@ -30,11 +30,11 @@
         expect(response).to render_template("rosters/new")
       end
 
-      it "redirects if the user doesn't belong to the organization" do
+      it "sends not found if the user doesn't belong to the organization" do
```

Comment:
Need feedback on this change @d12 
Rosters Controller sends a not found if the user can't be found so updated the test to match that.

Reply:
:+1: